### ISSUE TITLE
Compiling for ARM?

### ISSUE THREAD

**Issue by justchen1369** [mupen64plus/mupen64plus-core#594]
The installer throws a error when I try to compile for armv7

**Comment by Narann** [comment]
You should provide the error.

**Comment by enigma9o7** [comment]
[nemo@Sailfish source]$ cd mupen64plus-video-rice/
[nemo@Sailfish mupen64plus-video-rice]$ cd projects/
[nemo@Sailfish projects]$ cd unix
[nemo@Sailfish unix]$ make all NO_ASM=1 USE_GLES=1
Makefile:117: *** CPU type "aarch64" not supported.  Please file bug report at 'https://github.com/mupen64plus/mupen64plus-core/issues'.  Stop.

**Comment by akien-mga** [comment]
I can confirm that mupen64plus-core doesn't build on aarch64 out of the box, though Linux distros that package it have their own patches to add that support. See e.g. Fedora's (against version 2.5, doesn't apply out of the box on 2.5.9):

https://src.fedoraproject.org/rpms/mupen64plus/blob/master/f/mupen64plus-make-archs.patch

If there's interest upstream, it would be good to get these changes (or similar ones) integrated upstream so that Linux distros can build mupen64plus for their aarch64 targets without patches.

**Comment by akien-mga** [comment]
Actually, it seems aarch64 is now properly supported on mupen64plus-core and various plugins. But there's still some which fail when building the bundle:

```
make: Entering directory '/home/iurt/rpmbuild/BUILD/mupen64plus-bundle-src-2.5.9/source/mupen64plus-video-rice/projects/unix'
Makefile:117: *** CPU type "aarch64" not supported.  Please file bug report at 'https://github.com/mupen64plus/mupen64plus-core/issues'.  Stop.
```

**Comment by akien-mga** [comment]
And aarch64 support for `mupen64plus-video-rice` was added in https://github.com/mupen64plus/mupen64plus-video-rice/commit/772fdd9ecacf3669d7e50ea59ce4ef1f2c26b99b, which is not yet included in 2.5.9.

This issue could likely be closed as solved, unless you want to wait for a 2.5.10 that would include the patch for `mupen64plus-video-rice` too and make the default distribution aarch64-compatible.
